client/src/context/context.ts

```diff
@@ -1,24 +1,35 @@
-export interface ContextInterface {
+import { ContextMenuManager } from "./contextmenumanager";
+
+export type TContextConfig = {
```

Comment:
General comment: can we remove all the "T" in front of the Types? That's Hungarian notation, let's keep consistent with camel case